static/js/components/OrderSummary.js

```diff
@@ -36,20 +47,41 @@ class OrderSummary extends React.Component {
     return <p className="payment-explanation">{text}</p>;
   }
 
+  showAmount(description: string, amount: string): Array<React$Element<*>> {
+    return [
+      <Cell col={8} className="description mdl-cell--4-col-tablet mdl-cell--2-col-phone" key={description}>
+        {description}
+      </Cell>,
+      <Cell
+        col={2}
+        key={`${description}-amount`}
+        className="mdl-cell--2-col-tablet  mdl-cell--2-col-phone align-right">
+        <b className="amount">{amount}</b>
+      </Cell>
+    ];
+  }
+
   render() {
-    let { course, courseRun, checkout, checkoutStatus } = this.props;
+    let { course, courseRun, checkout, checkoutStatus, discount } = this.props;
+
+    let discountInfo;
+    if (discount){
```

Comment:
What if `discount` is `0`? We should do a more specific check here to avoid that problem